system/gluster_volume.py

```diff
@@ -329,11 +338,12 @@ def set_quota(name, directory, value):
 
         # create if it doesn't exist
         if volume_name not in volumes:
-            create_volume(volume_name, stripes, replicas, transport, cluster, brick_path)
+            create_volume(volume_name, stripes, replicas, transport, cluster, brick_path, force)
+            volumes = get_volumes()
```

Comment:
update volumes list after volume creation. Code below assumes the list is up-to-date